### ISSUE TITLE
@intlify/vue-i18n/no-html-messages does not report <strong>

### ISSUE THREAD

**Issue by jokay** [intlify/eslint-plugin-vue-i18n#271]
**Tell us about your environment**

- **ESLint version:** `8.2.0`
- **eslint-plugin-vue version:** `8.0.3`
- **eslint-plugin-vue-i18n version:** `1.0.1`
- **Node version:** `16.13.0`

**Please show your full configuration:**

```json5
'@intlify/vue-i18n/no-html-messages': 'error'
```

**What did you do?**

I let the linter do the magic run :wink:

#### en-US.json

```json
{
  "test": "<strong>gets not reported</strong>"
}
```

**What did you expect to happen?**

Should report an error.

**What actually happened?**

Does not report an error.

**Repository to reproduce this issue**

Not available.

**Comment by ota-meshi** [comment]
I can't reproduce your problem. Could you provide a repository that can reproduce the your problem?
I suspect there may be a problem with your repository settings.

**Comment by jokay** [comment]
Ok, no problem, will no longer bother with issues.

Issue closed.
